app/src/main/java/org/simple/clinic/storage/Migration_47_48.kt

```diff
@@ -0,0 +1,22 @@
+package org.simple.clinic.storage
+
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+
+@Suppress("ClassName")
+class Migration_47_48 : Migration(47, 48) {
+
+  override fun migrate(database: SupportSQLiteDatabase) {
+    database.execSQL("""
+    CREATE TABLE IF NOT EXISTS "Encounter" (
+    "uuid" TEXT NOT NULL,
+    "patientUuid" TEXT NOT NULL,
+    "encounteredOn" TEXT NOT NULL,
+    "createdAt" TEXT NOT NULL,
+    "updatedAt" TEXT NOT NULL,
+    "deletedAt" TEXT,
+    PRIMARY KEY("uuid")
+    )
+    """)
```

Comment:
Please check formatting?

Reply:
The SQL query string doesn't get auto-formatted.
We should maybe add a formatter for these as well.